js/task/ui.js

```diff
@@ -34,7 +34,7 @@ module.exports = (styleguide, gulp) => {
       // Pretend that the project is a package.
       const projectFiles = [
         path.join(styleguide.path.root(), 'package.json'),
-        path.join(styleguide.path.root(), 'styleguide/**/*.{hbs,json}')
+        path.join(styleguide.path.root(), 'styleguide/**/*.{hbs,json,md}')
```

Comment:
Add this extension to the watcher task as well, so that when a developer adds/updates a .md file, the watcher will pickup the file and build it.